packages/components/src/components/shared/FormText.tsx

```diff
@@ -11,39 +13,10 @@ type FormTextProps = {
 }
 
 /**
- * Fonts (TODO: replace with typography tokens)
+ * Fonts
```

Comment:
Don't think this comment is needed at all anymore? For a section that no longer exists.